src/apps__campaigns/models.py

```diff
@@ -0,0 +1,601 @@
+from django.db import models
+from django.forms import model_to_dict
+
+from _main_.utils.base_model import BaseModel
+from database.models import Event, Media, UserProfile, Community, Vendor
+from database.utils.common import get_json_if_not_none, get_summary_info
+from database.utils.constants import LONG_STR_LEN, SHORT_STR_LEN
+
+# Create your models here.
+
+class CampaignAccount(BaseModel):
```

Comment:
This was not in the spec.  Does a campaign have a single subdomain, or different subdomains per community?